tests/lib/snaps/snapctl-hooks/meta/hooks/configure

```diff
@@ -90,6 +90,14 @@ test_snapctl_foo_null() {
 	  fi
 }
 
+test_snapctl_unset() {
+	  echo "Unsetting an option"
+	  if ! snapctl set root.key2! ; then
+	  echo "snapctl set unexpectedly failed when un-setting root.key2"
```

Comment:
Indent looks funky

Reply:
I fixed this indent. There is another funny one at the bottom but the entire file is messed up so I don't want to do too much there.